tests/burnman_bfbt_test.prm

```diff
@@ -0,0 +1,145 @@
+# A test using adiabatic conditions and a gravity
```

Comment:
Is this a copy of burnman.prm? If yes, just replace the comment on top to say that this is testing bfbt, include the other prm file, and delete everything except the solver section here

Reply:
Also can you add the file of the screen output as well?